src/Components/ExperimentOne/ExperimentOne.js

```diff
@@ -1,12 +1,22 @@
+import ExperimentOneDisplayInformation from "./ExperimentOneDisplayInformation.js";
 import ExperimentOneHeader from "./ExperimentOneHeader.js";
-import ExperimentOneUserForm from "./ExperimentOneUserForm.js";
 
 const ExperimentOne = (props) => {
   return (
     <div>
       <ExperimentOneHeader />
-      <ExperimentOneUserForm />
-      <button onClick={props.isBackButtonClicked}>Back to Main Arena</button>
+      <ExperimentOneDisplayInformation />
+      <button
```

Comment:
Maybe make this a component so that you can reuse it with its styling easily?

Reply:
Yup, thats useful